### PR TITLE
#256 Rename document uploads to supportive documentation

### PR DESCRIPTION
**Pull request by slinke**
For #256 

Also did some styling to match the other wizard pages.
- [x] Rename 'Uploads' page of wizard
- [x] Add required document list

screen shot 2016-06-12 at 11 07 29 am

### CODE REVIEW

File: app/views/forms/uploads.html.erb [hunk 1]
```diff
@@ -2,16 +2,29 @@
...
+      <div class="row">
+        <div class="col-xs-12">
+          <h2>Supportive Documentation</h2>
+          <hr />
+        </div>
       </div>
-      <div class="actions">
-        <%= f.submit 'Upload Document' %>
-        <%= link_to 'Nevermind', class: 'button' %>
+      <div class="row">
```
Comment (scottjones) [inline app/views/forms/uploads.html.erb]: I don't think you need this extra `.row` div, as form-group is effectively the same.

**Comment by scottjones** [comment]
Looking good so far (added one minor comment).  Are you planning to also add the list of required files / instructions or something like that in there too?

File: app/views/forms/uploads.html.erb (hunk 1 above)
Comment (ghost) [inline app/views/forms/uploads.html.erb]: I only meant to show the separation of the h2 row with the row of form fields.

Comment (ghost) [inline app/views/forms/uploads.html.erb]: I'll remove it as I add the list of documents and ensure everything looks cool visually.

**Comment by ghost** [comment]
Yup.  Also adding the list.  Updated the screenshot.

File: app/views/forms/uploads.html.erb [hunk 2]
```diff
@@ -2,16 +2,57 @@
...
 <div class="row">
   <div class="col-md-8 col-sm-7">
-    <h2>Upload Grant Documents</h2>
-    <%= form_for(@upload, multipart: true) do |f| %>
-      <%= hidden_field(:upload, :user_id) %>
-      <%= hidden_field(:upload, :user_type) %>
-      <div class="field">
-        <%= f.file_field :file %>
+    <%= form_for(@upload, html: { class: 'new-upload form-horizontal', multipart: true }) do |f| %>
+      <div class="col-xs-12">
```
Comment (scottjones) [inline app/views/forms/uploads.html.erb]: Now that this isn't inside of a `.row` div, you can actually remove the `col-xs-12` also; it's not doing anything anymore (or you can add a wrapping `.row` around it).  The suggestion I had previously was intended to say that you didn't need to nest a `.form-group` inside of a `.row` as they are effectively the same and set up the row for the bootstrap grid.

File: app/views/forms/uploads.html.erb [hunk 3]
```diff
@@ -2,16 +2,57 @@
...
+        <h2>Supportive Documentation</h2>
+        <hr />
       </div>
-      <div class="actions">
-        <%= f.submit 'Upload Document' %>
-        <%= link_to 'Nevermind', class: 'button' %>
+      <div class="col-xs-12">
```
Comment (scottjones) [inline app/views/forms/uploads.html.erb]: Same comment here on the `.col-xs-12`

**Comment by scottjones** [comment]
Looking good!  Thanks!

**Pull request merged**